src/critic/app.py

```diff
@@ -70,3 +75,24 @@ def put_group(group_id: str):
             'message': 'OK (skeleton) — parsed and validated; no DB yet',
         }
     ), 200
+
+
+# Create monitor endpoint
```

Comment:
This comment seems misleading. This view runs due monitors; it doesn't create them. Right?

Guessing leftover from gen AI.